website/azurestack.erb

```diff
@@ -26,15 +26,15 @@
               <a href="#">Authentication</a>
               <ul class="nav nav-visible">
                 <li<%= sidebar_current("docs-azurestack-auth-service-principal-azure-cli") %>>
-                    <a href="/docs/providers/azurestack/guides/azure_cli.html">using the Azure CLI</a>
+                    <a href="/docs/providers/azurestack/auth/azure_cli.html">using the Azure CLI</a>
```

Comment:
Are these pages currently accessible or reliant on another PR? Neither `guides` nor `auth` work for me, but I'm coming into this PR blind, so I may be missing something.